app/models/health/vaccination.rb

```diff
@@ -190,10 +190,6 @@ def self.propagate_to_clients
       GrdaWarehouse::HealthEmergency::Vaccination.import(new_vaccinations) if new_vaccinations.present?
     end
 
-    def self.use_tsql_import?
```

Comment:
One of these was true right?  I feel like I recently had tried extricating it, which lead to this, my memory is shaky on why.

Reply:
Yes -- `Health::EpicBase` had it set to true.